README.md

```diff
@@ -121,6 +121,17 @@ This is a full example of a Drop-in integration that only accepts credit cards.
 </html>
 ```
 
+## Custom styling
+
+Customize the look and feel of Drop-in by using your own custom stylesheet.
+
+1. Copy the compiled CSS file to start with our Drop-in style as the foundation.
```

Comment:
Depends, if you clone down the project and build it, it'll exist at `dist/web/dropin/dev/css/dropin.css`. Otherwise, it'll exist on our cdn, but the version would need to be updated with every release.